### ISSUE TITLE
Can not open Options window with Firefox Nightly 93 build 2021-08-11

### ISSUE THREAD

**Issue by onemen** [onemen/TabMixPlus#62]
```
19:02:09.146 XML Parsing Error: undefined entity
Location: moz-nullprincipal:{2902ddbc-8191-47f0-b59b-099122e6dc7d}
Line Number 9, Column 9: {2902ddbc-8191-47f0-b59b-099122e6dc7d}:9:9
19:02:09.146
Uncaught Error: not well-formed XML
    parseXULToFragment chrome://global/content/customElements.js:566
    connectedCallback chrome://tabmixplus/content/preferences/prefs-ce.js:1091
    setDialog chrome://tabmixplus/content/preferences/preferences.js:722
    onload chrome://tabmixplus/content/preferences/preferences.xhtml:1
customElements.js:566:17
    parseXULToFragment chrome://global/content/customElements.js:566
    connectedCallback chrome://tabmixplus/content/preferences/prefs-ce.js:1091
    setDialog chrome://tabmixplus/content/preferences/preferences.js:722
    onload chrome://tabmixplus/content/preferences/preferences.xhtml:1
```

**Comment by onemen** [comment]
https://github.com/onemen/TabMixPlus/blob/3ce2e7779bbdd30945487bdd2256bceac62922df/addon/chrome/content/preferences/prefs-ce.js#L1091

@117649 , ```preferencesCmd2.label;``` was removed.
what do we use its value for?

**Comment by 117649** [comment]
> 
> 
> https://github.com/onemen/TabMixPlus/blob/3ce2e7779bbdd30945487bdd2256bceac62922df/addon/chrome/content/preferences/prefs-ce.js#L1091
> 
> @117649 , `preferencesCmd2.label;` was removed.
> what do we use its value for?
Looks like it is default title value for the dialog.

**Comment by onemen** [comment]
we don't need it we set our title in preferences.xhtml
```
<prefwindow type="prefwindow" class="prefwindow"
            id="TabMIxPreferences"
            windowtype="mozilla:tabmixopt"
            title="&page.header.title;"
```

**Comment by onemen** [comment]
Fixed by cffa07ad5c21a9b4783728c3779542138727376a

**Comment by 117649** [comment]
> 
> 
> we don't need it we set our title in preferences.xhtml
> 
> ```
> <prefwindow type="prefwindow" class="prefwindow"
>             id="TabMIxPreferences"
>             windowtype="mozilla:tabmixopt"
>             title="&page.header.title;"
> ```

Yes indeed, It was carried from old XBL code.
